File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 3/src/Ex1.java
//Importando Scanner
import java.util.Scanner;

public class Ex1{
	
	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner ent = new Scanner(System.in);
		
		System.out.println("Maior e menor entre 15 valores\n\n");
		
		//vars
		float maior, menor, num;
		int i;
		
		//Entrada
		System.out.print("Insira o  1° número: ");
		num = ent.nextFloat();
		
		maior = num;
		menor = num;
		
		for( i=2; i<=15; i++ )
		{
			//Entrada
			System.out.print("Insira o " + i + "° número: ");
			num = ent.nextFloat();

			//Processamento
			if( num > maior )
				maior = num;
			else if ( num < menor )
				menor = num;

		}
		
		//Saída
		System.out.println("Maior número: " + maior);
		System.out.println("Menor número: " + menor);
		
		
		
	}
}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 3/ex1.c
#include <stdio.h>

main()
{
    printf("Maior e menor entre 15 valores\n\n");

    //vars
    int i, num, maior, menor;

    printf("Insira um numero inteiro (1): ");
    scanf("%i", &num);

    //Processamento
    maior = num;
    menor = num;

    for (i=2; i<=15; i++)
    {
        if( num > maior )
        {
            maior = num;
        }
        else if ( num < menor )
        {
            menor = num;
        }

        printf("Insira um numero inteiro(%d): ", i);
        scanf("%d", &num);
    }

    //Saída
    printf("\nMenor numero: %d\n", menor);
    printf("Maior numero: %d\n", maior);

}
